README.md

```diff
@@ -219,7 +219,7 @@ Flags:
   -v, -- count                   Set log level, higher is more verbose
   -h, --help                     help for iq
   -a, --iq-application string    Specify public application ID for request (required)
-  -x, --iq-server-url string     Specify Nexus IQ Server URL (default "http://localhost:8070")
+  -x, --iq-server-url string     Specify Nexus IQ Server URL (default "http://host.docker.internal:8070")
```

Comment:
Also curious about how this helps? (Faster, etc?)
I typically don't run IQ in a docker container, but I'm that last to claim my usage pattern is typical. ;)

Would the command `ahab config` (which allows you to set a local default value for the `iq-server-url`) be another solution?

Reply:
I don't run IQ in a docker container either - it is usually is on my team's host(laptop) - hence the default being the host.docker.internal - which says, my IQ is on my laptop, not in the container.